cmp/fd175594-0a7e-443f-ae71-6126adbf106e/component.lp

```diff
@@ -0,0 +1,37 @@
+(librepcb_component fd175594-0a7e-443f-ae71-6126adbf106e
+ (name "Regulator with Enable")
+ (description "A voltage regulator with enable pin.")
+ (keywords "regulator,ldo,dc-dc,voltage")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2019-01-28T19:26:25Z)
+ (deprecated false)
+ (category 30643e64-b131-4eb5-b956-ceb3cb02413a)
+ (schematic_only false)
+ (default_value "")
+ (prefix "U")
+ (signal c12bcf97-b5fa-4533-ad05-b3b5d9bd3e2d (name "IN") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 3a9f594b-8d53-4cf4-816a-a85d265730e7 (name "OUT") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 0abb84c9-f452-4ae1-a6a3-c3cf31e3157c (name "GND") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 9b86c706-c378-4a19-85d3-2135a0ace920 (name "EN") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
```

Comment:
Actually I'm not sure if there exist regulators which don't need the enable pin to be connected. But probably it's better to risk a false-positive ERC warning instead of a false-negative, thus I would declare this signal as required:

```suggestion
  (required true) (negated false) (clock false) (forced_net "")
```

Reply:
Yeah, you can always connect the pin to GND or VCC.